utils.py

```diff
@@ -475,6 +477,36 @@ def get_human_readable_time_string(time_msec):
         '%B %d %H:%M:%S', time.gmtime(python_utils.divide(time_msec, 1000.0)))
 
 
+def create_string_from_largest_unit_in_timedelta(timedelta_obj):
+    """Given the timedelta object, find the largest non zero time unit and
+    return that value, along with the time unit, as a string.
+
+    Args:
+        timedelta_obj: A datetime timedelta object. Datetime timedelta
```

Comment:
```suggestion
        timedelta_obj: datetime.timedelta. Datetime timedelta
```

We should keep the formatting consistent here with other docstrings.

Reply:
Changed, sorry!